include/grpcpp/alts_context.h

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPCPP_ALTS_CONTEXT_H
+#define GRPCPP_ALTS_CONTEXT_H
+
+#include <grpcpp/impl/codegen/security/auth_context.h>
+
+#include <memory>
+// might need forward declaration
+#include "src/core/tsi/alts/handshaker/alts_tsi_handshaker.h"
+#include "src/proto/grpc/gcp/altscontext.upb.h"
+
+namespace grpc {
+
+enum SecurityLevel {
```

Comment:
I'd rather not having so many definition of Security Level

Let's use the security level definition in https://github.com/grpc/grpc/pull/21215. @yihuazhang  do you have an ETA on that PR?

Reply:
The SecurityLevel is a re-definition of the one in upb: https://github.com/grpc/grpc/blob/0e9a73696d86ddd5137bf8bf16c76ee36fc76ba7/src/core/ext/upb-generated/src/proto/grpc/gcp/transport_security_common.upb.h#L30-L34
This seems incompatible with the enum defined in Yihua's PR.
My intention here is to try to avoid anything with name "grpc_gcp_XXX" delivered directly to users. But since this is just an enum, maybe we could just reuse the grpc_gcp_SecurityLevel?

Reply:
Thanks for the review @jiangtaoli2016! I've added another utility function for getting AltsContext, and move the security_level to grpc_security_constants.h. Please let me know if you find anything else. Thanks again for the review!